javascript/react/intro-react/why-use-react.md

```diff
@@ -18,24 +18,22 @@ aspects:
   - introduction
 
 ---
-# Why use **React**?
+# Why use React?
 
 ---
 ## Content
 
-**React** is considered a great choice for building the **UI** of web applications - be it simple *SPAs*[1] or
-large projects with numerous authors due to the **self-containing** nature of its *components*.
+**React** is considered a great choice for building the **UI** of web applications - be it simple *SPAs*[1] or large projects with numerous authors due to the **self-containing** nature of its *components*.
 
 In addition to the previously stated reasons developers are eager to adopt this library, **React** comes with many more interesting features such as:
 
 - great *performance* as a result of the **Virtual DOM** diff algorithm
-- **React**'s syntax is simple. Unlike other popular `JS` frameworks with specific syntax, **React** makes use of pure `JS`
+- **React**'s syntax is simple. Unlike other popular JS frameworks with specific syntax, **React** makes use of pure JS
 - easier debugging with **out-of-the-box tools** such as `React Chrome Extension`
-- **React** is good for *Search Engine Optimisation* when employing server-side rendering
+- **React** is good for *Search Engine Optimization* when employing server-side rendering
```

Comment:
Hmmmm not really a benefit. In fact, SEO is more difficult because of React, not vice-versa. We should remove this.